src/Microsoft.DotNet.Build.Tasks/ParsePackageNames.cs

```diff
@@ -31,7 +31,7 @@ public override bool Execute()
                     Log.LogWarning("PackageDrop does not exist - '{0}'", packageDrop);
                     continue;
                 }
-                IEnumerable<ITaskItem> packages = Directory.GetFiles(packageDrop)?.Select(f => new TaskItem(f));
+                IEnumerable<ITaskItem> packages = Directory.GetFiles(packageDrop)?.Select(f => new TaskItem(Path.GetFileNameWithoutExtension(f)));
```

Comment:
Thanks Wes.  For some reason I was thinking that Directory.GetFiles returned null for an empty file set, which I think would be valid.  Actually taking a moment to looking at the API, clearly shows that it returns an empty array.  I'll update this and keep your feedback in mind.